src/transport/DisabledLogTransport.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace LogHero\Client;
+
+
+class DisabledLogTransport implements LogTransportInterface {
+    protected $logBuffer;
+
+    public function __construct(LogBufferInterface $logBuffer) {
+        $this->logBuffer = $logBuffer;
+    }
+
+    public function submit($logEvent) {
+        $this->logBuffer->push($logEvent);
+        if ($this->logBuffer->needsDumping()) {
+            $this->flush();
```

Comment:
Why is there a call to method doing nothing?

Reply:
The idea is to define three different types of transports. This one is meant to do nothing whenever a flush is triggered (flush needs to be triggered elsewhere). This class does also serve as a base class for the other two transport types (sync and async) which overwrite the flush() method with the appropriate logic. Another way to implement this would be adding a new abstract class which abstracts away the implementation of the flush() method and creating subclasses with different implementations. In this case I simply wanted to save such an abstract class.

Reply:
I can't really see the reason to save this class. If it helps to make things more clear you should go for it. In my opinion, it's crucial to write clear and understandable code, especially for open source projects. Otherwise, it's harder to maintain, extend and more likely that bugs may be introduced.